crates/forge/tests/e2e/running.rs

```diff
@@ -843,42 +843,61 @@ fn should_panic() {
     let snapbox = test_runner();
 
     let output = snapbox.current_dir(&temp).assert().code(1);
+
     assert_stdout_contains!(
```

Comment:
I know you prob copied the output you got (and it's completely fine), just FYI: this macro doesn't check the order in which lines are printed